lib-android-robolectric/src/main/java/com/devinci/lib/robolectric/AttributeSetBuilder.java

```diff
@@ -0,0 +1,38 @@
+package com.devinci.lib.robolectric;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import java.util.ArrayList;
+import java.util.List;
+import javax.annotation.Nonnull;
```

Comment:
It required regular Android library configuration so it would pass lint checks. It also failed on some dexing task if I recall because of a lot of robolectric's dependencies.